### PR TITLE
Update forked action to AppTweak (BACK-473)

### PR DESCRIPTION
**Pull request by Dakad**
## What does this PR do?

Setup repo to be used as github-action for apptweak-utils-ruby.

After the merge, it will be tagged with `v2.4.0` or `v2.3.0+apptweak.1.0.0` to stay valid with Semver

## Checklist

- [x] My code follows the code style of this project.
- [ ] I have run tests locally (manual tests and otherwise).
- [ ] This has been tested on staging.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] My change includes a database modification
  - [ ] I have tested my database modification in staging.
- [ ] My changes require changes in other components/squads/teams
  - [ ] I already did the changes in the other components or notified the responsible people that the changes need to be done
  - [ ] The needed changes are already deployed or ready to be deployed

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -22,4 +20,4 @@ cd "${WORKING_DIRECTORY:-.}"
 echo "Building the gem"
 find . -name '*.gemspec' -maxdepth 1 -exec gem build {} \;
```
Comment (renatolond) [inline entrypoint.sh]: Isn't the idea to remove the maxdepth here for monorepos?

Comment (Dakad) [reply entrypoint.sh]: I tried the command it while being at the project root. It didn't work.
image

THe right exec command would be something like ` gem build -C frontend_utils apptweak-frontend-utils.gemspec`

Comment (Dakad) [reply entrypoint.sh]: I had to update the exec  with `dirname {}` and `basename {}`

Comment (Dakad) [reply entrypoint.sh]: Fixed

**Review by renatolond — APPROVED** [review]

**Pull request merged**